shell/platform/windows/windows_lifecycle_manager.cc

```diff
@@ -48,7 +48,7 @@ bool WindowsLifecycleManager::WindowProc(HWND hwnd,
     // send a request to the framework to see if the app should exit. If it
     // is, we re-dispatch a new WM_CLOSE message. In order to allow the new
     // message to reach other delegates, we ignore it here.
-    case WM_CLOSE:
+    case WM_CLOSE: {
```

Comment:
The braces were added as the C++ compiler complained about potential jumping to uninitialized values